packages/cli/test/functional/test_site/expected/testCodeBlocks.html

```diff
@@ -284,8 +284,8 @@ <h3 id="testing-site-nav"><span id="testing-site-nav" class="anchor"></span>Test
 </span><span><span class="hljs-tag"><span class="highlighted">&lt;/</span><span class="hljs-name">foo</span>&gt;</span>
 </span></code></pre>
         <p><strong><code class="line-numbers hljs inline no-lang" v-pre>highlight-lines</code> attr with line-part syntax should highlight only at specified substring</strong></p>
-        <pre><code class="line-numbers hljs xml" v-pre><span><span class="hljs-tag">&lt;<span class="hljs-name">foo</span>&gt;</span>
-</span><span>  <span class="hljs-tag">&lt;<span class="hljs-name">bar</span> <span class="hljs-attr highlighted">type</span>=<span class="hljs-string">&quot;name&quot;</span>&gt;</span>goo<span class="hljs-tag">&lt;/<span class="hljs-name">bar</span>&gt;</span>
+        <pre><code class="line-numbers hljs xml" v-pre><span><span class="hljs-tag"><span><span class="highlighted"></span>&lt;</span><span class="hljs-name"><span><span class="highlighted"></span>foo</span></span><span><span class="highlighted"></span>&gt;</span></span><span><span class="highlighted"></span>
```

Comment:
I think so?

Reply:
Yes, it looks ok cosmetically.

Let's find out the cause at least though, its somewhat strange that removing the `boundEnd !== boundStart` check actually removes this diff. I intuitively expected the opposite since the check should make the output more concise.